tyr-core/src/main/java/org/jboss/tyr/check/TemplateChecker.java

```diff
@@ -128,9 +128,11 @@ public String checkPR(JsonObject payload) throws InvalidPayloadException {
                     format.getDescription().getOptionalRows()));
         }
 
-        if (format.getCommit() != null) {
-            commitMessagesCheck.setRegex(format.getCommit());
-            checks.add(commitMessagesCheck);
+        if (!configuration.disableCommitCheck()){
+            if (format.getCommit() != null) {
```

Comment:
also doesn't need an inner if statement

Reply:
Had to merge them, because the tests started to fail.